### PR TITLE
Updates for contributions and where to discuss

### PR DESCRIPTION
**Pull request by fubuloubu**
For #332

Missing link for contributing (couldn't find it)

Cheetah

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
 ## Contributing
 * See Issues above, and feel free to submit your own issues or PRs if you discover a solution
-* Discussions and questions on [Gitter](https://gitter.im/ethereum/viper)
+* Further discussions and questions [chat on gitter](https://gitter.im/bethereum/viper)
```
Comment (Dexaran) [inline README.md]: `bethereum/viper` in link address is a typo.

You should use `ethereum/viper`

**Comment by fubuloubu** [comment]
Duh, thanks!

**Comment by DavidKnott** [comment]
@fubuloubu Here's the link for contributing http://eth-viper.readthedocs.io/en/latest/contributing.html

**Comment by fubuloubu** [comment]
I added the link. I will make a note of adding the docs main page plus the latest (changelog?), but can we merge this and I can do that in a separate PR? Since that is a larger undertaking to revise the README

**Comment by DavidKnott** [comment]
@fubuloubu I agree 100% merging it now.

**Pull request merged**